### ISSUE TITLE
Timeout too short

### ISSUE THREAD

**Issue by danburd** [evuraan/yellowShoes#2]
Thank you for this program.

I am very close to getting it working on my Raspberry Pi 4, but I believe the timeout is too short for my system. It takes longer than 18 seconds for nrsc5 to lock on to a signal and play audio. As a result, when I use yellowShoes, it generates an error log that looks like the beginning of an nrsc5 console output about to begin playing a stream. I think it needs just a few more seconds for audio to begin.

I believe the yellowShoes timeout is currently set at 18 seconds. When I play nrsc5 without yellowShoes on the same device, it takes 20+ seconds, but it does seem to work. I found someone else who identified this issue [here](https://forum.silicondust.com/forum/viewtopic.php?t=75512), but I don't know how to increase the timeout so that I can use yellowShoes.

Thank you again.

**Comment by evuraan** [comment]
Thank you for taking the time to file this report. I will look into making `playerTimeout`  a tunable setting. Until then, can you please try changing it by hand and see if it works better for you?

```bash
user@lego:~/git/yellowShoes$ sed -i 's/18000/30000/g' static/page.html 
```
In the example above, I've changed it from `18000` to `30000` milliSeconds. 

Please give this a try and report back!

**Comment by danburd** [comment]
Thank you! This worked wonderfully. It was just what I was looking for.

I wasn't savvy enough to find where in the program the timeout was coded.

It's up to you whether to make the timeout a tunable setting - I'm satisfied regardless.

Thanks again!

**Comment by evuraan** [comment]
Glad to hear it worked for you! 

I've gone ahead and made and published the changes in branch `tunableTimeO`:

https://github.com/evuraan/yellowShoes/tree/tunableTimeO 

Changes:
* playerTimeout raised to 30 seconds 
* set `playerTimeout` section in settings - 

![playerTimeout](https://user-images.githubusercontent.com/39205936/128801131-799cdc6f-4508-49b4-adc4-0abf767743af.png)

Please give it a try, and let me know how it goes. I am looking to merge to main in a few days of scream tests.

**Comment by danburd** [comment]
It worked for me - I replaced the html file as you described and I can now access the menu item where I can change the timeout. Thank you again!

**Comment by evuraan** [comment]
Changes merged to main, published `1.09d-ui-1.5f` release.
